### ISSUE TITLE
Do not use reserved words as variable names in examples

### ISSUE THREAD

**Issue by kosz** [cloudhead/node-static#147]
Maybe you know about this. I've added 'use strict' to the example code and tried to run it 

``` javascript
'use strict';
var static = require('node-static');

var fileServer = new static.Server('./public');

require('http').createServer(function (request, response) {
    request.addListener('end', function () {
        fileServer.serve(request, response);
    }).resume();
}).listen(8080);
```

Error : 

```
  var static = require('node-static');
      ^^^^^^
SyntaxError: Unexpected strict mode reserved word
    at Module._compile (module.js:439:25)
```

Could confuse the hell out of some people.

**Comment by NataliaLKB** [comment]
:+1:

**Comment by brettz9** [comment]
Closed by #227 (renamed to `statik`)
